src/web3py/extensions/lido_validators.py

```diff
@@ -174,7 +174,9 @@ def get_lido_validators_by_node_operators(self, blockstamp: BlockStamp) -> Valid
     @lru_cache(maxsize=1)
     def get_module_validators_by_node_operators(self, module_id: StakingModuleId, blockstamp: BlockStamp) -> ValidatorsByNodeOperator:
         """Get module validators by querying the KeysAPI for the module keys"""
-        raise NotImplementedError()
+        # TODO: Re-evaluate the implementation, sub-optimal solution.
```

Comment:
very sub-optimal, I'd say. we have 1% CSM validators and 99% from other modules, even on testnet. so I'd suggest to do the plain request(s) to KeysAPI here to get the validators + operators by the module address